battle-game-functions-jpmaatz/src/script.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+// TODO - write your code here.
+const randomDamage = () => Math.floor(Math.random()*10)+1 
+
+const chooseOption = (opt1,opt2) => Math.random() < 0.5 ? opt1:opt2
+
+function attackPlayer (health){
+    return health - randomDamage;
+}
+
+const logHealth = (player,health) => console.log("" + player + " health " + health + "!")
+
+const logDeath = (winner,loser) => console.log("" + winner + " defeated " + loser + "!")

```

Comment:
Not sure why the return string starts with empty double quotations. I don't think that's needed. Also consider using string templating / interpolation with backticks:

```
`${player} health ${health}!`

`${winner} defeated ${loser}!`
```

Reply:
Using plus signs to concatenate strings is a valid strategy but in my opinion, for simpler calculated strings, interpolation is easier to read right away.